### PR TITLE
Update CLI text

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description

https://jira.mesosphere.com/browse/DCOS-13971

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by joel-hamill** [comment]
@tamarrow see https://github.com/dcos/dcos-docs/pull/1064/files#diff-024c3ad7ecffcba5ea3f9efa8bb29877 and https://github.com/dcos/dcos-docs/pull/1064/files#diff-1c9fb4d15b766118fd34da54a05d44bb.

File: 1.8/usage/cli/index.md [hunk 1]
```diff
@@ -37,21 +37,21 @@ Get detailed command description with 'dcos <command> --help'.
...
+#### DCOS_CONFIG
+Specifies the path to the DC/OS configuration file. By default, this variable is set to `DCOS_CONFIG=/<home-directory>/.dcos/dcos.toml`. For example, if you moved your DC/OS configuration file to `/home/jdoe/config/` you can specify this command:
 
 ```bash
 export DCOS_CONFIG=/home/jdoe/config/dcos.toml
 ```
     
-#### `DCOS_SSL_VERIFY` 
-Indicates whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). By default, this is variable is set to `true`. Setting this environment variable is equivalent to setting the `core.ssl_config` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
+#### DCOS_SSL_VERIFY (required)
```
Comment (tamarrow) [inline 1.8/usage/cli/index.md]: not required, since you can use the config variable `core.ssl_verify`

File: 1.8/usage/cli/index.md [hunk 2]
```diff
@@ -37,21 +37,21 @@ Get detailed command description with 'dcos <command> --help'.
...
+Specifies whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). You must set this variable manually. Setting this environment variable is equivalent to setting the `dcos config set core.ssl_verify` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
```
Comment (tamarrow) [inline 1.8/usage/cli/index.md]: verify SSL certificates... (`true` or `false`). or set the path to the SSL certificates (`path to certificates`

File: 1.8/usage/cli/index.md [hunk 3]
```diff
@@ -65,64 +65,66 @@ For example, to set the log level to warning:
...
-**pagination** Indicates whether to paginate output. By default this is set to `True`. For example, to set to false:
+#### pagination
+Indicates whether to paginate output. By default this is set to `True`. For example, to set to false:
 
 ```bash
 dcos config set core.pagination False
 ``` 
 
-**reporting** Indicates whether to report usage events to Mesosphere. By default this is set to `True`. For example, to set to false:
+#### reporting
```
Comment (tamarrow) [inline 1.8/usage/cli/index.md]: we should remove, we dont use this anymore

File: 1.8/usage/cli/index.md [hunk 4]
```diff
@@ -65,64 +65,66 @@ For example, to set the log level to warning:
...
+Indicates whether to report usage events to Mesosphere. By default this is set to `True`. For example, to set to false:
 
 ```bash
 dcos config set core.reporting False
 ```
     
-**ssl_verify** Indicates whether to verify SSL certs for HTTPS or path to certs. By default this is set to `False`. For example, to set to true:
+#### ssl_verify
+Indicates whether to verify SSL certs for HTTPS or path to certs. By default this is set to `False`. For example, to set to true:
```
Comment (tamarrow) [inline 1.8/usage/cli/index.md]: by default it is set to True

File: 1.9/cli/index.md [hunk 5]
```diff
@@ -31,27 +31,26 @@ Available DC/OS commands:
...
+#### DCOS_CONFIG
+Specifies the path to the DC/OS configuration file. By default, this variable is set to `DCOS_CONFIG=/<home-directory>/.dcos/dcos.toml`. For example, if you moved your DC/OS configuration file to `/home/jdoe/config/` you can specify this command:
 
 ```bash
 export DCOS_CONFIG=/home/jdoe/config/dcos.toml
 ```
     
-#### `DCOS_SSL_VERIFY` 
-Indicates whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). By default, this is variable is set to `true`. Setting this environment variable is equivalent to setting the `core.ssl_config` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
+#### DCOS_SSL_VERIFY (required)
```
Comment (tamarrow) [inline 1.9/cli/index.md]: not required

File: 1.9/cli/index.md [hunk 6]
```diff
@@ -65,61 +64,64 @@ For example, to set the log level to warning:
...
-**reporting** Indicates whether to report usage events to Mesosphere. By default this is set to `True`. For example, to set to false:
+#### reporting
+Indicates whether to report usage events to Mesosphere. By default this is set to `True`. For example, to set to false:
 
 ```bash
 dcos config set core.reporting False
 ```
     
-**ssl_verify** Indicates whether to verify SSL certs for HTTPS or path to certs. By default this is set to `False`. For example, to set to true:
+#### ssl_verify
```
Comment (tamarrow) [inline 1.9/cli/index.md]: default is true

File: 1.9/cli/index.md [hunk 7]
```diff
@@ -65,61 +64,64 @@ For example, to set the log level to warning:
...
-**pagination** Indicates whether to paginate output. By default this is set to `True`. For example, to set to false:
+#### pagination
+Indicates whether to paginate output. By default this is set to `True`. For example, to set to false:
 
 ```bash
 dcos config set core.pagination False
 ``` 
 
-**reporting** Indicates whether to report usage events to Mesosphere. By default this is set to `True`. For example, to set to false:
+#### reporting
```
Comment (tamarrow) [inline 1.9/cli/index.md]: can remove this section

File: 1.9/cli/index.md [hunk 8]
```diff
@@ -31,27 +31,26 @@ Available DC/OS commands:
...
+Specifies the path to the DC/OS configuration file. By default, this variable is set to `DCOS_CONFIG=/<home-directory>/.dcos/dcos.toml`. For example, if you moved your DC/OS configuration file to `/home/jdoe/config/` you can specify this command:
 
 ```bash
 export DCOS_CONFIG=/home/jdoe/config/dcos.toml
 ```
     
-#### `DCOS_SSL_VERIFY` 
-Indicates whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). By default, this is variable is set to `true`. Setting this environment variable is equivalent to setting the `core.ssl_config` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
+#### DCOS_SSL_VERIFY (required)
+Specifies whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). You must set this variable manually. Setting this environment variable is equivalent to setting the `dcos config set core.ssl_verify` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
```
Comment (tamarrow) [inline 1.9/cli/index.md]: verify SSL certificates... (true or false). or set the path to the SSL certificates (path to certificates)

**Review by tamarrow-zz — APPROVED** [review]
thanks @joel-hamill, a few nits.

File: 1.8/usage/cli/index.md (hunk 3 above)
Comment (joel-hamill) [inline 1.8/usage/cli/index.md]: @tamarrow as of 1.8?

File: 1.9/cli/index.md (hunk 8 above)
Comment (joel-hamill) [inline 1.9/cli/index.md]: Changed to `Indicates whether to verify SSL certificates or set the path to the SSL certificates.`

Comment (joel-hamill) [inline 1.9/cli/index.md]: Where do users set the path?

File: 1.8/usage/cli/index.md (hunk 2 above)
Comment (joel-hamill) [inline 1.8/usage/cli/index.md]: Changed to `Indicates whether to verify SSL certificates or set the path to the SSL certificates.`

File: 1.7/usage/cli/index.md [hunk 9]
```diff
@@ -90,7 +90,7 @@ The configuration settings are stored in the `dcos.toml` file. You can modify th
     dcos config set core.reporting False
     
 
-**ssl_verify** Indicates whether to verify SSL certs for HTTPS or path to certs. By default this is set to `False`. For example, to set to true:
+**ssl_verify** Indicates whether to verify SSL certs for HTTPS or path to certs. By default this is set to `True`. For example, to set to true:
```
Comment (sascala) [inline 1.7/usage/cli/index.md]: I think the example should set this to False, if the default is True

File: 1.8/usage/cli/index.md [hunk 10]
```diff
@@ -65,64 +65,59 @@ For example, to set the log level to warning:
...
-
-```bash
-dcos config set core.reporting False
-```
     
-**ssl_verify** Indicates whether to verify SSL certs for HTTPS or path to certs. By default this is set to `False`. For example, to set to true:
+#### ssl_verify
+Indicates whether to verify SSL certs for HTTPS or path to certs. By default this is set to `True`. For example, to set to true:
 
 ```bash
```
Comment (sascala) [inline 1.8/usage/cli/index.md]: example should set it to False

File: 1.9/cli/command-reference/dcos-config/dcos-config-set.md [hunk 11]
```diff
@@ -12,7 +12,7 @@ Add or set a DC/OS configuration properties. Here are the available properties.
 | core.dcos_url         | This property contains the public master IP of your DC/OS cluster.|
 | core.mesos_master_url | This property specifies the Mesos master URL.|
 | core.pagination       | This property indicates whether to paginate output. |
-| core.ssl_verify       | This property specifies whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). |
+| core.ssl_verify       | This property Indicates whether to verify SSL certificates or set the path to the SSL certificates. |
```
Comment (sascala) [inline 1.9/cli/command-reference/dcos-config/dcos-config-set.md]: Indicates should be lower case

**Pull request merged**